templates/vue/src/services/TapestryAPI.js

```diff
@@ -1,13 +1,16 @@
 import axios from "axios"
 import Helpers from "../utils/Helpers"
+import { data } from "./wp"
+
+const { apiUrl, nonce, postId } = data
 
 class TapestryApi {
   /**
    *
    * @param {Number} postId
    */
   constructor(postId) {
-    axios.defaults.headers.common["X-WP-Nonce"] = wpData.nonce
+    axios.defaults.headers.common["X-WP-Nonce"] = nonce
```

Comment:
Does this not need to be `data.nonce`?

Reply:
No - `nonce` was destructured around line 5 I believe.